sdk/python/feast/client.py

```diff
@@ -368,9 +343,7 @@ def archive_project(self, project):
         if self._project == project:
             self._project = FEAST_DEFAULT_OPTIONS[CONFIG_PROJECT_KEY]
 
-    def apply_entity(
-        self, entities: Union[List[EntityV2], EntityV2], project: str = None
-    ):
+    def apply_entity(self, entities: Union[List[Entity], Entity], project: str = None):
```

Comment:
should this be `apply_entities`?

Reply:
If we change this, then we should also change `apply_feature_table`. Do we prefer plural?